### ISSUE TITLE
[Bug]: Northstar Menu items are not visible on keyboard focus in High Contrast

### ISSUE THREAD

**Issue by davyrwoo** [microsoft/fluentui#26727]
### Library

React Northstar / v0 (@fluentui/react-northstar)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
    Memory: 1.33 GB / 15.84 GB
  Browsers:
    Chrome: 109.0.5414.120
    Edge: Spartan (44.22621.1105.0), Chromium (108.0.1462.76), ChromiumDev (111.0.1660.2)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codesandbox.io/s/mjbj60?module=/example.js

### Bug Description

1. Select the High Contrast theme
2. Tab to the menu item

## Actual Behavior
Menu item text is not visible 

## Expected Behavior
Menu item text should remain visible

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by micahgodbolt** [comment]
![image](https://user-images.githubusercontent.com/1434956/219179983-962176f8-886e-44a4-b51e-f60c13bf4de7.png)
